portfolio/src/main/webapp/timeline.js

```diff
@@ -0,0 +1,44 @@
+const dates = ['', 'August 2018', 'October 2018', 'May 2019 - August 2019', 'August 2019', 'May 2020 - August 2020', ''];
```

Comment:
You can make this a bit easier to use by grouping the events in the same object instead of linking by index:

```javascript
const events = [
{ date: 'july 1', event: 'got a dog'},
{ date: 'july 2', event: 'took dog for walk'},
{ date: 'july 3', event: 'dog is good'},
```

And then you can access the fields
```javascript
events[0].date; // july 1
events[0].event; // got a dog
events.length // there are 3 events
```

Reply:
changed